app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java

```diff
@@ -22,18 +30,65 @@
 @RunWith(AndroidJUnit4.class)
 public class HomeScreenActivityTest {
 
-    @Rule
-    public ActivityScenarioRule<HomeScreenActivity> HomeScreenTestRule = new ActivityScenarioRule<>(HomeScreenActivity.class);
+    /*@Rule
```

Comment:
why do you comment this rule ?

Reply:
I wanted to launch the Homescreen myself after having signed in or out to make sure that it would pass a potential authentification check when the Homescreen activity was created.